hack/templates/resources/90-shoot.yaml.tpl

```
@@ -320,6 +320,12 @@ spec:
   #     SomeKubernetesFeature: true
   #   runtimeConfig:
   #     scheduling.k8s.io/v1alpha1: true
+  #-#-# requires TokenRequest feature gate
+  #-#-# See https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+  #   serviceAccountConfig:
+  #     issuer: "some issuer"
```

It's a recommended that Isser be an URL starting with `https://`

```suggestion
  #     issuer: https://${value("spec.dns.domain", value("metadata.name", "johndoe-" + cloud) + "." + value("metadata.namespace", "garden-dev") + ".example.com")}
```